CONTRIBUTING.md

```diff
@@ -0,0 +1,201 @@
+# Contributing
+
+The Archway project welcomes all contributors.
+
+This document serves as a guide for contributing to the Archway project. It provides an overview of the processes and policies established for quality collaboration and coordination. Please familiarize yourself with these guidelines and feel free to propose changes to this document for improvement.
+
+## 1. IDE
+
+### 1.1 Software Dependencies
+
+The following software should be installed on the target system:
+
+- The Go Programming Language (https://go.dev)
+- Git Distributed Version Control (https://git-scm.com)
+- Docker (https://www.docker.com)
+- GNU Make (https://www.gnu.org/software/make)
+
+### 1.2 Fork & Clone
+
+1. [Fork the Archway repository](https://github.com/archway-network/archway);
+2. [Sync your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) to keep it up to date;
+3. Clone your fork to an appropriate directory in your local environment;
+
+```sh
+git clone https://github.com/${YOU}/archway.git
+```
+
+### 1.3 Install and Configure IDE
+
+Download and install an IDE such as Visual Studio Code on your system.
+
+_Contributors may use whatever IDE they are comfortable with, however this guide will focus on VSCode._
+
+Add the following extensions:
```

Comment:
Nice to have:

vscode allows [Profiles](https://code.visualstudio.com/docs/editor/profiles) which can be imported and the profiles can have all the extension and settings configuration presetup. We could publish a `archway.code-profile` doc which can be directly imported to set up their work env.